test/invariants/ExampleInvariantUsage.t.sol

```diff
@@ -236,13 +236,19 @@ contract ExampleInvariantUsage is Test, OpcodesDebug, CoreInvariants {
             exactOutData
         );
 
+        uint256[] memory amounts = new uint256[](3);
```

Comment:
(not to fix right now) Created a task to test all instructions with more wide liquidity and swap amounts ranges